### ISSUE TITLE
Provide 5 minute introductions to the technology.

### ISSUE THREAD

**Issue by fire** [V-Sekai/many_bone_ik#7]
**Describe the bug**
EWBIK can be hard to understand.

**To Reproduce**

Try hard to understand the technology document and make noises.

**Expected behavior**
Easy to understand for multiple proficiency levels.

Provide 5 minute introductions to the technology.

**Comment by fire** [comment]
# Human joints

Provide standard joints kusudamas.

##  immovable are fixed joints

## slightly movable joints

slightly movable joints of the vertebrae of the spine are good examples.

## Freely movable joints

Freely movable joints enable all parts of the joint to smoothly move against each other. Examples include joints like the knee and shoulder.
> 
> 1. Ball and socket joint. Permitting movement in all directions, the ball and socket joint features the rounded head of one bone sitting in the cup of another bone. Examples include your shoulder joint and your hip joint.
> 2. The hinge joint is like a door, opening and closing in one direction, along one plane. Examples include your elbow joint and your knee joint.
> 3. The Condyloid joint allows movement, but no rotation. Examples include your finger joints and your jaw.
> 4. The pivot joint is characterized by one bone that can swivel in a ring formed from a second bone. Examples are the joints between your ulna and radius bones that rotate your forearm, and the joint between the first and second vertebrae in your neck.
> 5. Gliding joint only permits limited movement, it’s characterized by smooth surfaces that can slip over one another. An example is the joint in your wrist.
> 6. Saddle joint. Although the saddle joint does not allow rotation, it does enable movement back and forth and side to side. An example is the joint at the base of your thumb.
> 

Reference from: 

https://www.healthline.com/health/saddle-joint#takeaway

**Comment by fire** [comment]
![Uploading image.png…]()

**Comment by fire** [comment]
![image](https://user-images.githubusercontent.com/32321/202057196-70b0aac2-a21e-4ac6-b572-c832dc4c241f.png)

**Comment by fire** [comment]
tl;dr hinges and sockets are the most common joints.
